### DISCUSSION TITLE
It should be possible to re-order comments

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#2442] [Ideas]
This isn't something we're going to do right away, but it's something to consider. If we want the comments area to tell a story, sometimes you want to re-order them.

**Comment by jonbell-lot23** [comment]
March 15, 2022 update:

This isn't an 80% case, but it's definitely a pain point in our current system for some people.

**Comment by jaril** [comment]
<img width="1366" alt="image" src="https://user-images.githubusercontent.com/15959269/179088520-10de78c4-305d-4262-b749-8444eefc9e9c.png">

**Reply by markerikson** [reply]
Yeah, this would be _really_ useful for me to tell the story of how I worked backwards from the error message through a dozen different files:

https://app.replay.io/recording/logpoint-02js--2d7e7544-33f3-4e39-b82d-b4f7d0b2daf5?point=22067261651551680457036465793139134&time=10896.647887323943&hasFrames=true#

**Reply by jonbell-lot23** [reply]
100%! We should absolutely do this, just a matter of timing.

But I am quite eager to do this feature for this very reason.
